src/styles/theme/index.js

```diff
@@ -6,9 +6,9 @@ export const theme = {
         unit: 4
     },
     breakpoints: {
-        xs: 576,
-        sm: 768,
-        md: 992,
+        xs: 0,
```

Comment:
Perchè xs è stato portato a 0?
Continuo a non capire questi breakpoints? Sembrano andare contro il principio di "mobile-first"

Reply:
Per i breakpoints mi sono ispirato a MUI ([vedi qui](https://material-ui.com/customization/breakpoints/)). Il mobile first rimane, dato che ovunque vengono usate delle media query, le dimensioni all'infuori di queste media query (quelle di default) sono sempre riferite al mobile.

Questo sistema di breakpoint, abbinato ai componenti `Grid` ed `Hidden`, velocizza notevolmente lo sviluppo di interfacce responsive, e mantiene consistency appunto tra i vari breakpoint nel sito (prima erano hardcodati in ciascun componente)

Reply:
Grazie per la spiegazione, ora mi sono chiari i breakpoints - preferisco, dove necessario, utilizzare standard riconosciuti, non capivo da dove arrivassero i valori scelti.

Ottimo lavoro con `Grid` e `Hidden` 👍